client/extensions/woocommerce/app/store-stats/store-stats-navigation/style.scss

```diff
@@ -0,0 +1,40 @@
+.followers-count {
+	.button {
+		margin-right: 16px;
+	}
+}
```

Comment:
Followers Count is going to be moved anyways
https://github.com/Automattic/wp-calypso/issues/14560